### PR TITLE
[java/en] Add Switching on String Example to Java

### PR DESCRIPTION
**Pull request by cschermerhorn**
Added a simple example on how to switch strings in Java 7 and above.

### CODE REVIEW

File: java.html.markdown [hunk 1]
```diff
@@ -275,6 +276,23 @@ public class LearnJava {
                      break;
         }
         System.out.println("Switch Case Result: " + monthString);
+        
+        //Starting in Java 7 and above, switching Strings works like this:
```
Comment (vendethiel) [inline java.html.markdown]: please add a space before the comment (and after //)

Comment (cschermerhorn) [inline java.html.markdown]: Added, thanks for providing feedback.

**Comment by levibostian** [comment]
Can you merge in master, please?

**Comment by cschermerhorn** [comment]
Merged newest master into my pull request.

**Pull request merged**